js/scripts.js

```diff
@@ -1,94 +1,116 @@
 //IIFE
 let a24Repository = (function() {
-	let movieList = [
-		{
-			name: 'The Witch',
-			year: '2015',
-			length: 92,
-			genre: [
-				'Drama',
-				'Fantasy',
-				'Horror'
-			],
-			URL: 'https://de.web.img3.acsta.net/c_310_420/pictures/16/03/21/12/29/404212.jpg'
-		},
-		{
-			name: 'Lady Bird',
-			year: '2017',
-			length: 94,
-			genre: [
-				'Comedy',
-				'Drama'
-			],
-			URL: 'https://i0.wp.com/teaser-trailer.com/wp-content/uploads/Lady-Bird-New-Film-poster.jpg'
-		},
-		{
-			name: 'Moonlight',
-			year: '2016',
-			length: 111,
-			genre: [
-				'Drama'
-			],
-			URL: 'https://www.cineworld.co.uk/xmedia-cw/repo/feats/posters/HO00004174.jpg'
-		}
-	];
-	// function for adding a movie
-	function add(movie) {
-		if (typeof movie === 'object' && 'name' in movie && 'year' in movie && 'length' in movie && 'genre' in movie) {
-			movieList.push(movie);
+	let pokemonList = [];
+	let apiUrl = 'https://pokeapi.co/api/v2/pokemon/?limit=20';
+
+	function loadingMessage() {
+		let loadImg = document.createElement('image');
+		loadImg.src = 'https://www.sketchup.com/sites/www.sketchup.com/modules/license_wizard_green/images/loading.gif';
```

Comment:
One way to debug is to open [dev tools](https://nira.com/chrome-developer-tools/#:~:text=From%20the%20Chrome%20menu%3A,web%20page%20you're%20on.) by holding down the `fn + F12` on the element on the browser. 
- In this case we have `document.createElement('image')` that should have been `document.createElement('img')`.

Reply:
Ah, sure – just tag-names